### PR TITLE
Collections

### PR DESCRIPTION
**Pull request by lysyjan**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -813,3 +827,15 @@ export namespace Tours {
...
 }
+
+export namespace Collection {
+	export interface Collection {
+		id: number;
+		nameLong: string;
+		nameShort: string | null;
+		description: string | null;
+		tags: Places.Tag[];
+		places: Promise<Places.Place[]>;
```
Comment (NeosinneR) [inline index.d.ts]: This should be without `Promise`, just `Places.Place[]`

Comment (lysyjan) [reply index.d.ts]: fixed

File: index.d.ts [hunk 2]
```diff
@@ -813,3 +817,15 @@ export namespace Tours {
 	export type ToursQueryDirection = 'asc' | 'desc';
 	export type ToursQuerySortBy = 'price' | 'rating' | 'top_sellers' ;
 }
+
+export namespace Collection {
```
[the remark this answers is not in the document]
Comment (fekecrad) [reply index.d.ts]: It's not a big deal, mergeable

[the remark this answers is not in the document]
Comment (lysyjan) [reply index.d.ts]: It's a mistake, I'll fix it

**Comment by lysyjan** [comment]
fixed

**Pull request merged**